.github/workflows/build_package.yml

```diff
@@ -35,4 +35,8 @@ jobs:
           OPENWEATHERMAP_API_KEY: "${{ secrets.OPENWEATHERMAP_API_KEY }}"
           ANTHROPIC_API_KEY: "${{ secrets.ANTHROPIC_API_KEY }}"
           COHERE_API_KEY: "${{ secrets.COHERE_API_KEY }}"
+          AZURE_OPENAI_API_KEY: "to-be-filled"
+          AZURE_API_VERSION: "to-be-filled"
+          AZURE_DEPLOYMENT: "to-be-filled"
+          AZURE_OPENAI_ENDPOINT: "https://camel.openai.azure.com/"
```

Comment:
Node: need an administrator of camel.ai to help configure the relevant keys.

Reply:
There is currently no account, so set it to TODO @Wendong-Fan